play/src/front/Components/Input/InputCheckbox.svelte

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
     export let id: string | undefined = undefined;
+    export let datatestid: string | undefined = undefined;
```

Comment:
```suggestion
    export let dataTestId: string | undefined = undefined;
```
Let's keep the same name as Input